### PR TITLE
Optionally play a "ding" sound when receiving a message

### PR DESCRIPTION
**Pull request by YanisBft**
Comes with a toggle command

This PR has been tested on a server, everything works as expected

### CODE REVIEW

**Comment by JRoy** [comment]
Has this been tested on every mc version (1.8.8-1.16.2)

If I recall, the sound enum was changed

**Comment by YanisBft** [comment]
I will test asap

**Comment by YanisBft** [comment]
> Has this been tested on every mc version (1.8.8-1.16.2)
> If I recall, the sound enum was changed

Tested in 1.8.8, 1.9.4, 1.12.2 and 1.16.2, and fixed in 1.8.8 :)

File: Essentials/src/com/earth2me/essentials/messaging/SimpleMessageRecipient.java [hunk 1]
```diff
@@ -147,6 +150,19 @@ public MessageResponse onReceiveMessage(IMessageRecipient sender, String message
...
         sendMessage(tl("msgFormat", sender.getDisplayName(), tl("meRecipient"), message));
 
+        // Play a "ding" sound when receiving a message
+        if (user != null && !user.isIgnoreMsgSound()) {
+            Sound dingSound;
+
+            if (VersionUtil.getServerBukkitVersion().isHigherThanOrEqualTo(VersionUtil.v1_9_R01)) {
+                dingSound = Sound.ENTITY_EXPERIENCE_ORB_PICKUP;
+            } else {
+                dingSound = Sound.valueOf("ORB_PICKUP");
```
Comment (montlikadani) [inline Essentials/src/com/earth2me/essentials/messaging/SimpleMessageRecipient.java]: You can literally make sound configurable and use `bool ? : `

Comment (YanisBft) [reply Essentials/src/com/earth2me/essentials/messaging/SimpleMessageRecipient.java]: True, just didn't to make it easier to read

**Review by JRoy — CHANGES_REQUESTED** [review]
Your toggle command doesn't actually store that value in the user config, please look at other toggle commands to figure out how to do that

**Comment by mdcfe** [comment]
This particular feature seems fairly niche in EssentialsX, and given server owners who use this will probably want more control over how this is implemented, I'm not convinced this is worth adding to EssentialsX directly. This can be achieved easily through a third-party plugin using the API.

**Pull request closed without merging**